changelog.R

```diff
@@ -266,8 +278,7 @@ chglog <- tribble(
 )
 
 
-output$changelog <- renderHtmlTableWidget({ 
-  chglog %>% 
-    addHtmlTableStyle(align="l") %>% 
-    htmlTableWidget(rnames=FALSE) 
-})
+output$changelog <- renderDataTable({
+  chglog2 <- chglog
+}, rownames=FALSE, escape=FALSE)
```

Comment:
keeping to tidyverse styleguide, be sure you're linting your code, or typing it with spaces between operators. In RStudio, you can do cctrl-shift-a after selecting a bit of code and it will lint it for you